docker/synapse_sytest.sh

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env bash
+
+set -x
+
+if [ -e "/test/run-tests.pl" ]
```

Comment:
why is this this `/test/run-tests.pl` rather than `./run-tests.pl` ? The inconsistency makes me wonder if I'm misunderstanding what is going on. Which is entirely plausible, given my level of docker fu.